packages/changed-elements-react/src/widgets/comparisonJobWidget/components/VersionCompareManageNamedVersions.tsx

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import "./styles/ComparisonJobWidget.scss";
+
+
+export interface ManageNamedVersionsProps {
```

Comment:
You don't actually need to export React component props type. It is rarely needed and can be accessed using built-in React type `ComponentProps`, e.g. `ComponentProps<ManageNamedVersions>`.

Nit: remove extra empty line above.